contracts/SubdomainRegistrar.sol

```diff
@@ -112,9 +112,9 @@ contract SubdomainRegistrar is RegistrarInterface {
      * @param name The name to transfer.
      * @param newOwner The address of the new owner.
      */
-    function transfer(string name, address newOwner) public owner_only(keccak256(name)) {
-        bytes32 label = keccak256(name);
-        OwnerChanged(keccak256(name), domains[label].owner, newOwner);
+    function transfer(string memory name, address payable newOwner) public owner_only(keccak256(abi.encode(name))) {
```

Comment:
You can simply cast `name` to `bytes memory` - no need to use `abi.encode` here.